main.js

```diff
@@ -194,5 +243,16 @@ $showCart.addEventListener('click', (e) => {
     }
     render(cartArray)
   }
-  console.log(e.target)
+})
+
+const $submitCheckout = document.querySelector("[data-view='checkout']")
+$submitCheckout.addEventListener('click', (e) => {
```

Comment:
In general, we do not listen for `click` events when dealing with forms. Instead, handle a `submit` event. But be sure to prevent the default behavior of form submissions.